testng-curl/azure-pipelines-1.yml

```diff
@@ -0,0 +1,35 @@
+# Maven
```

Comment:
Jenkins and aure-pipeline code inside same folder creates a confusion, lets have two folders, one with Jenkins and one with azure, so that users dont get confused with multiple CI tools used here